src/utils/makeReactNativeConfig.js

```diff
@@ -52,6 +52,29 @@ type DEPRECATEDWebpackConfigFactory = (
   WebpackConfig
 ) => WebpackConfig | WebpackConfig;
 
+const DEFAULT_INCLUDED_NODE_MODULES = [
+  'react',
+  '@react-navigation',
+  '@react-native-community',
+  '@expo',
+  'pretty-format',
+  'haul',
+  'metro',
+];
+
+/**
+ * Constructs a RegExp for the default JavaScript processing rule
+ */
+const getJavaScriptExcludeRegex = (
```

Comment:
Nit: I would rename this to something else. It's not strictly creating an exclude regex though it might be used as such.